components/package-template/src/etc/clp-config.yml

```diff
@@ -2,7 +2,9 @@
 ## workers.
 ## - This path will be exposed inside the container, so symbolic links to files
 ##   outside this path will be ignored.
-#input_logs_directory: "/"
+#logs_input:
+#  type: "fs"
+#  directory: "/"
```

Comment:
Unrelated this PR. We might want to confirm the behavior if the directory is not "/".

For example, if the directory is set to "/a/b/c", and we want to compress "/a/b/c/d", do we type full path or relative path for compress.py?

Reply:
@Eden-D-Zhang do you mind quickly verify this?